udaykalra/portfolio/target/portfolio-1/style.css

```diff
@@ -1,9 +1,144 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+.bottomRow
+{
+	padding-top: 30px;
 }
-
-#greeting-container {
-  margin-top: 20px;
+.center
+{
+	margin: auto;
+	text-align: center;
 }
+.hero-image
+{
+	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+ url('hero.jpg');
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: cover;
+	height: 50%;
+	position: relative;
+}
+.hero-text
+{
+	color: white;
+	font: 'Roboto', sans-serif;
+	font-size: 40px;
```

Comment:
Shorthand properties are encouraged:
https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

It looks like there's one for font:
https://www.w3schools.com/css/css_font_shorthand.asp

Reply:
This has been implemented in latest push.